backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "yeoman-ui",
-	"version": "1.0.14",
+	"version": "1.0.15",
 	"displayName": "Application Wizard",
 	"publisher": "SAPOS",
```

Comment:
when do we plan to change the publisher to "SAPOSS" ?

Reply:
@tomer-epstein  Changing publisher is still in the discussion, there is a meeting on Thursday when and how to do it gracefully with minimal breaking